portfolio.html

```diff
@@ -17,8 +17,8 @@
     </nav>
   </header>
```

Comment:
- [ ] Please make sure you desktop looks same as in the design, the desktop version has links to the various sections of the page, and the hamburger button is removed. Kindly fix that. See image below. 

***The design***
![lockD1](https://user-images.githubusercontent.com/65852621/134504740-4a73633c-ca9d-4998-9d0b-76d9903c756c.png)

***Your implementation***
![locki1](https://user-images.githubusercontent.com/65852621/134504778-bebaa9ec-4372-4371-a5cc-263d8b9cead9.png)